# Fix safebrowsing build on macOS ("disk_image_type_sniffer_mac" is set only if safe_browsing_mode == 1)

--- chrome/common/safe_browsing/BUILD.gn	2018-12-13 23:09:09.000000000 +0300
+++ b/chrome/common/safe_browsing/BUILD.gn	2018-12-13 23:10:15.000000000 +0300
@@ -73,11 +73,11 @@
     ":file_type_policies",
   ]
 
-  if (is_mac) {
-    deps += [ ":disk_image_type_sniffer_mac" ]
-  }
-
   if (safe_browsing_mode == 1) {
+    if (is_mac) {
+      deps += [ ":disk_image_type_sniffer_mac" ]
+    }
+
     sources = [
       "binary_feature_extractor.cc",
       "binary_feature_extractor.h",
